assets/sass/patterns/molecules/social-links.scss

```diff
@@ -26,11 +26,40 @@
   }
 }
 
+.github-link__item {
+  display: flex;
+  @include margin(0 0 37);
+  flex-flow: row wrap;
+  align-items: center;
+  justify-content: center;
+  min-height: 24px;
+  color: $color-text;
+
+  svg path {
+    fill: $color-text;
+  }
+  &:hover svg path {
+    fill: $color-primary;
+  }
+
+  .github-link--text {
+    @include margin(2 0 0 7);
+
+    &:hover {
```

Comment:
Think this is meant to apply to the parent?

Reply:
Have added parent class